README.md

```diff
@@ -101,6 +101,12 @@ To see full configuration types see [template.lua](./lua/leetcode/config/templat
     ---@type boolean
     logging = true,
 
+    ---@type boolean
+    toggle_console_on_run = false,
+
+    ---@type boolean
+    toggle_console_on_submit = false,
```

Comment:
```suggestion
    show_on_runcode = false,
```
toggle would mean that when you have the console already opened it would close it on runcode.
Also move it to the `console` table. I know readme states its for appearance only so i will have to change that